samples/visual-indicator/nodejs/README.md

```diff
@@ -0,0 +1,128 @@
+---
+page_type: sample
+description: This sample app shows how to send visual indicator in meetings using targeted notifications API.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "23/05/2023 13:38:26 PM"
+urlFragment: officedev-microsoft-teams-samples-visual-indicator-nodejs
+---
+
+## Visual Indicator In Meeting Notiofication
+
+This sample illustrates how to send visual indicator notification on tab icon in meetings.
```

Comment:
This sample illustrates how to send visual indicator notification on tab icon in Teams meetings.

Reply:
fixed as per comment